tutorial/practicals/REQUIREMENTS.md

```diff
@@ -0,0 +1,44 @@
+# Software environment for the practical examples
+
+* Linux OS or virtual machine (e.g. [Oracle VM Virtual Box](
+  https://www.virtualbox.org/)).
+
+* Fortran compiler supporting [Fortran 2003 standard.](
+  https://gcc.gnu.org/wiki/GFortranStandards#Fortran_2003)
+  The `Makefile`s in the examples are based on the freely available [GNU
+  Compiler Collection (GCC)](https://gcc.gnu.org/). LFRic code is routinely
+  built with GCC 6.1.0, Intel Fortran Compiler 17.0.x and Cray Fortran
+  Compilers 8.4.3 and 8.7.0. It has been proven to work with Intel 19.0.x
+  and the latest GCC 10.2.0.
+
+* [Python](https://www.python.org/) environment. Note that for this
+  tutorial either [Python 2](https://www.python.org/download/releases/2.0/)
+  or [Python 3](https://www.python.org/download/releases/3.0/) environments
+  are fine (plotting scripts require `numpy` and `matplotlib` packages in
+  either case).
+
+* Latest PSyclone release (currently 1.9.0). For installation please refer to
```

Comment:
I just noticed this. Should this be the current git release, not the latest official one?

Reply:
Ah! We have a bit of a clash here as I've added a `tutorial/practicals/README.md` in my 933_ branch. We definitely shouldn't be using the word 'release' anywhere :-)

Reply:
Shall I delete this document then, @arporter?

Reply:
Yes, if you wouldn't mind - I think that's the simplest solution.

Reply:
Deleted